tutorial.lisp: add information about enable and disable modes on the manual

pdelfino opened the pull request:
Hi,

As discussed in a previous private email, here is a suggestion to insert the commands `enable-mode` and `disable-mode` on the manual. I tried to keep the style similar to the other parts. Not sure if I did manage to do that, though. I can iterate the content. Thanks!

aartaka commented on source/tutorial.lisp:
```diff
@@ -176,7 +176,8 @@ removed from the input, they are also removed from the existing bookmark.")
 list of all bookmarks."))
-
+   (:h3 "Enable and Disable modes")
+   (:p "Unlike other modes, " (:code "enable-mode") " represents an upper level of abstraction having a meta profile. It allows the user to apply other modes (e.g., " (:code "nosound-mode") " or " (:code "dark-mode") ") to multiple buffers at once. Conversely, it is possible to revert this action executing " (:code "disable-mode")" while choosing exactly the same buffers and the same mode previou ...
```
Wow, that's a long one!

Try to make sure that every line is less than 100 chars long — it's more readable this way. Don't be afraid to insert line breaks — they all will disappear once this code is converted to HTML.

Regarding content: it looks fine, but the length on the line makes it really inconvenient to read, so sonething could've escaped my attention :)

pdelfino replied:
Sorry! I was editing this in Emacs with two windows like a vertical split. I did not notice this fact because of the word wrapping. I will pay intention on it and update the PR.

Ambrevar commented on source/tutorial.lisp:
```diff
@@ -176,7 +176,13 @@ removed from the input, they are also removed from the existing bookmark.")
-
+   (:h3 "Enable and Disable modes")
+   (:p "Unlike other modes, " (:code "enable-mode") " represents an upper 
+level of abstraction having a meta profile. It allows the user to apply other
```
What do you mean with "meta profile"?

Ambrevar commented on source/tutorial.lisp:
```diff
@@ -176,7 +176,13 @@ removed from the input, they are also removed from the existing bookmark.")
+level of abstraction having a meta profile. It allows the user to apply other
+ modes (e.g., " (:code "nosound-mode") " or " (:code "dark-mode") ") to multiple
+ buffers at once. Conversely, it is possible to revert this action executing "
+ (:code "disable-mode")" while choosing exactly the same buffers and the same 
```
Remove the leading space in the above lines.

pdelfino replied:
Sorry, not sure if I get it. 

Are you referring to remove the line breaks I inserted after the suggestion made by @aartaka on this PR yesterday?
Should I restore it to the way it was?

jmercouris replied:
Notice that there is whitespace on line 182 before 'modes' and on 183 before 'buffers' etc.

pdelfino replied:
This how I see things in emacs:
![image](https://user-images.githubusercontent.com/7698207/122265877-90456e00-ceaf-11eb-9864-36fec012bd91.png)

Please, take a look at the cursor. Should this space exist between "a" and the "line break"? 

The word on the next line called "mode" does not have a whitespace by its left side (in emacs).

Ambrevar commented on source/tutorial.lisp:
```diff
@@ -176,7 +176,13 @@ removed from the input, they are also removed from the existing bookmark.")
+   (:h3 "Enable and Disable modes")
+   (:p "Unlike other modes, " (:code "enable-mode") " represents an upper 
+level of abstraction having a meta profile. It allows the user to apply other
+ modes (e.g., " (:code "nosound-mode") " or " (:code "dark-mode") ") to multiple
```
Prefer "such as" to the more cryptic "e.g.".

Ambrevar commented:
> By "meta" I mean that enable-**mode** is _a mode_ built to activate _other modes._ 
> Does it sound inappropriate?

enable-mode is not a mode, it's a command.

pdelfino commented:
> By "meta" I mean that enable-**mode** is _a mode_ built to activate _other modes._ Does it sound inappropriate?
> enable-mode is not a mode, it's a command.

Sorry. You are correct. I am pushing a similar text. This time making clear enable-mode is a command and without the 'meta' part.

pdelfino commented:
Hey, @jmercouris ,

After we applied `fill-paragraph` and  used `whitespace-mode`, I added and committed the change. When I pushed, one of the tests started to fail. I tried reading the message but it says something about "global history". I do not know why this is happening.

Before talking to you, I removed a line that I thought could be an extra space. Initially, I thought this could be the root of the problem. So I reverted the commit (even though it seems harmless deleting a blank line).

Despite reverting the commit, the test still fails. Could you help me understand what happened?

jmercouris commented:
The test is unrelated to your changes, our CI is having some problems currently, I wouldn't worry about it!
